.github/workflows/devcontainer.yml

```diff
@@ -84,6 +84,7 @@ jobs:
           --env CONTAINER_WORKSPACE_FOLDER=/workspaces/MLOS \
           --env LOCAL_WORKSPACE_FOLDER=$(pwd) \
           --workdir /workspaces/MLOS \
+          --add-host host.docker.internal:host-gateway \
```

Comment:
Mimics the changes to `devcontainer/docker-compose.yml` so we can know which endpoint to talk to for tests.

See Also: `mlos_bench/tests/services/remote/ssh/conftest.py`